### PR TITLE
Update Chainer at a Glance

### PR DESCRIPTION
**Pull request by Crissman**
* Improve resolution of images--addresses #5130 
* Add glance.py code to examples directory
* Change Glance docs to refer to glance.py code instead of copy/paste segments
* Convert from `import chainer` to `import chainer as ch` style
* Use chainer.Sequential for defining network model
* Move to one-line format for setting up optimizer

### CODE REVIEW

File: examples/glance/glance.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+train_iter = ch.iterators.SerialIterator(train, 100)
+test_iter = ch.iterators.SerialIterator(
+    test, 100, repeat=False, shuffle=False)
+
+
+# Network definition
+def MLP(n_units, n_out):
+    layer = ch.Sequential(L.Linear(n_units), F.relu)
+    model = layer.repeat(1)
```
Comment (toslunar) [inline examples/glance/glance.py]: `repeat(2)` to make the network equivalent to the old one.

**Comment by Crissman** [comment]
Appveyor error appears unrelated.

**Comment by Crissman** [comment]
@toslunar Both fixed. Thanks!

**Comment by toslunar** [comment]
Jenkins, test this please.

**Comment by toslunar** [comment]
These `literalinclude` include some empty lines above and below.  Is it OK?

**Pull request merged**